thamos/cli.py

```diff
@@ -129,6 +129,10 @@
     "python_version": "Python version",
 }
 
+THOTH_SEARCH_UI_BASE_URL = os.getenv(
+    "THOTH_SEARCH_UI_URL", "https://thoth-station.ninja/search/"
```

Comment:
This will not work on client's machines unless they provide `THOTH_SEARCH_UI_URL` environment variable explicitly. It might be a good idea to obtain Thoth Search UI URL as reported by the API service in the response headers. You can do so by checking headers in `api_discovery` method:

https://github.com/thoth-station/thamos/blob/125c04ecdcfa3e2e80e65b78ef8416129efc80cb/thamos/config.py#L466

Reply:
Right, thanks! 💯 I used the CLI configuration to get the URL from the API headers: https://github.com/thoth-station/thamos/pull/1083/commits/bee58b9b393d67f0e2230b5e6b01c65d06bb7a44